.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+1. Julia Version
+
+2. Operating system
+
+3. Minimum working example
```

Comment:
I think this might be better:

```mkd
## To Reproduce

1. Julia Version (i.e. output of `julia -v`):



2. Operating system (Mac, Linux, Windows):



3. Minimum working code example that led to bug:



```